### PR TITLE
W-10815221 Reference Guide AS2 Connector 6.3.0

### PR DESCRIPTION
**Pull request by sms-eceluci**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: as2/6.3/modules/ROOT/pages/as2-connector-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+			</tls:context>
+		</http:listener-connection>
+	</http:listener-config>
+
+	<configuration-properties doc:name="Configuration properties" doc:id="b0baf25d-17ef-4d10-bb24-2c5b25e6289c" file="app.properties" />
+	<as2-mule4:listener-config name="AS2_Connector_Listener_ForwardVAN" doc:name="AS2 Connector Listener config" doc:id="59fd60b5-92f4-4532-9b03-41be1e8c27bc" httpListenerConfig="HTTP_Listener_config" securityLevel="SIGNED_ENCRYPTED" >
+		<as2-mule4:self-config as2Name="forward-van" x509Alias="forward-van" email="support@forward-van.com" />
+		<as2-mule4:partner-config as2Name="myth-as2" x509Alias="myth-as2" email="support@mythical.com" />
+		<as2-mule4:key-store-config keystorePassword="test" keystorePath="as2/forward-van.p12" privateKeyPassword="test" />
+	</as2-mule4:listener-config>
```
Comment (vijayang) [inline as2/6.3/modules/ROOT/pages/as2-connector-examples.adoc]: Please remove this part. It was in the app for some POC. It need not be in the docs.

File: as2/6.3/modules/ROOT/pages/as2-connector-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+		<as2-mule4:self-config as2Name="#[vars.as2Sender]" x509Alias="#[vars.as2Sender]" email="support@forward-van.com" />
+		<as2-mule4:partner-config as2Name="#[vars.as2Receiver]" x509Alias="#[vars.as2Receiver]" email="support@mythical.com"/>
+		<as2-mule4:requester-config subject="EDI" messageIntegrityCheckAlgorithm="SHA256" mdnMessageIntegrityCheckAlgorithm="SHA256" encryptionAlgorithm="DES_EDE3" requestReceipt="SIGNED_REQUIRED"/>
+		<as2-mule4:keystore-config keystorePassword="test" keystorePath="as2/forward-van.p12" privateKeyPassword="test" />
+	</as2-mule4:send-config>
+	<flow name="forward-van-as2-simulatorFlow" doc:id="7908ee11-bda5-40b9-a3bc-53bcea5cc96c" >
+		<as2-mule4:as2-listener doc:name="Forward VAN AS2 Listener" doc:id="39f3e8e1-c75f-489f-b148-509d4638c1d6" config-ref="AS2_Connector_Listener_ForwardVAN" path="/forward-van/as2-receive"/>
+		<logger level="INFO" doc:name="Logger" doc:id="5f8dcd82-64c4-4188-9197-f0237e7e35d2" message="Received AS2 message from Mythical's Partner Manager"/>
+		<logger level="INFO" doc:name="IDOC Payload from Mythical to Forward VAN" doc:id="d4499996-9c51-4080-8494-9da219a7ff31" message="#[payload]"/>
+	</flow>
```
Comment (vijayang) [inline as2/6.3/modules/ROOT/pages/as2-connector-examples.adoc]: Please remove this part also.

**Review by vijayang — CHANGES_REQUESTED** [review]

**Review by sms-asalanova — DISMISSED** [review]

**Review by vijayang — DISMISSED** [review]

File: as2/6.3/modules/ROOT/pages/as2-connector-reference.adoc [hunk 3]
```diff
@@ -0,0 +1,827 @@
...
+| Dedicated mode - Partner Config - x509 Alias a| String |  Certificate, private key alias, or both in the keystore. Required if using signing or encryption. |  |
+| Dedicated mode - Partner Config - email a| String |  Email address used as the HTTP `From` header when sending AS2 messages. |  | x
+|Shared mode - Partners |  Enumeration, one of:
+
+* Expression
+* Inline
+| Indicates whether to add partner information by using an expression or by entering values inline.| Expression | x
+| Shared mode - Partners - AS2 Partner Name a| String |  AS2 system identifier used for AS2-From and AS2-To headers. |  | x
+| Shared mode - Partners - x509 Alias a| String |  Certificate, private key alias, or both in the keystore. Required if using signing or encryption. |  |
+| Shared mode - Partners - email a| String |  Email address used as the HTTP 'From' header when sending an AS2 message. |  | x
```
Comment (sathya0) [inline as2/6.3/modules/ROOT/pages/as2-connector-reference.adoc]:
```suggestion
| Shared mode - Partners - email a| String |  Email address used as the HTTP `From` header when sending an AS2 message. |  | x
```

File: as2/6.3/modules/ROOT/pages/as2-connector-reference.adoc [hunk 4]
```diff
@@ -0,0 +1,827 @@
...
+If you select *Mdn edicated mode*, check *Partner config* (mandatory) and configure the partner using the Mdn dedicated mode fields in this table.
++
+* Mdn shared mode
++
+Enables you to configure a single AS2 listener endpoint to receive MDNs from multiple partners.
++
+If you select *Mdn shared mode*, uncheck *Partner config* (mandatory) and configure each partner using the Mdn shared mode fields in this table. |  | x
+| Mdn dedicated mode - Partner Config - AS2 Partner Name a| String |  AS2 system identifier used for AS2-From and AS2-To headers. |  | x
+| Mdn dedicated mode - Partner Config - x509 Alias a| String |  Certificate, private key alias, or both in the keystore. Required if using signing or encryption. | Dedicated mode|
+| Mdn dedicated mode - Partner Config - email a| String |  Email address used as the HTTP 'From' header when sending MDNs. |  | x
```
Comment (sathya0) [inline as2/6.3/modules/ROOT/pages/as2-connector-reference.adoc]:
```suggestion
| Mdn dedicated mode - Partner Config - email a| String |  Email address used as the HTTP `From` header when sending MDNs. |  | x
```

File: as2/6.3/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+
+== Changes
+
+=== 6.3
+
+AS2 Connector 6.3 contains the following changes:
+
+* You can now configure a single AS2 listener endpoint to receive messages from multiple partners.
++
+To implement this feature, a new tab named *Partners* was added to the *As 2 listener* and *As 2 mdn listener* global elements. This tab contains fields that were were moved from the *Partner Config* section of the *Geneneral* tab, as well as some new fields.
```
Comment (sathya0) [inline as2/6.3/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc]:
```suggestion
To implement this feature, a new tab named *Partners* was added to the *As2 listener* and *As2 mdn listener* global elements. This tab contains fields that were moved from the *Partner Config* section of the *General* tab, as well as some new fields.
```

Comment (dhirsch1) [reply as2/6.3/modules/ROOT/pages/as2-connector-upgrade-migrate.adoc]: Sadly, "AS 2 mdn listener" is the name in the GUI.

**Review by sathya0 — APPROVED** [review]

**Pull request merged**